FAQ.md

```diff
@@ -105,17 +105,41 @@ But you can also use a crate like [ctrlc](https://docs.rs/ctrlc/latest/ctrlc/).
 ## How to use `log` or `tracing` as default log backend
 
 * **log**, add the feature flag `logger_log` to the dependency in `Cargo.toml`
-    ```toml
-    iceoryx2 = { version = "0.1.0", features = ["logger_log"]}
-    ```
+  ```toml
+  iceoryx2 = { version = "0.1.0", features = ["logger_log"]}
+  ```
 * **tracing**, add the feature flag `logger_tracing` to the dependency in
   `Cargo.toml`
-    ```toml
-     iceoryx2 = { version = "0.1.0", features = ["logger_tracing"]}
-    ```
+  ```toml
+   iceoryx2 = { version = "0.1.0", features = ["logger_tracing"]}
+  ```
+
+## Supported log levels
+
+iceoryx2 support different log levels
+`Trace, Debug, Info, Warning, Error, Fatal`
 
 ## How to set the log level
 
+iceoryx2 gets its default logging level from `.cargo/config.toml` , but this can
+be over-ridden by environment variable `IOX2_LOG_LEVEL` the developer can set
+this environment variable with one of the supported levels.
+
+`export IOX2_LOG_LEVEL=Trace`
+
+then in the main.rs call the function to read the environment variable
+
+```rust
+use iceoryx2::prelude::*
+
+// ...
+
+set_log_level_from_env_or_default();
+```
+
+Developers can also set the logging level from within the code which will take
+precedence over the environment variable:
+
 ```rust
 use iceoryx2::prelude::*
```

Comment:
What do you think of adding all three options to set the log level in the same code block? Maybe by adding an additional comment above each option?